docs/source/rpc.rst

```diff
@@ -97,7 +97,11 @@ applications can always explicitly move the input tensors to CPU on the caller
 and move it to the desired devices on the callee if necessary.
 
 .. warning::
-  TorchScript support in RPC is experimental and subject to change.
+  TorchScript support in RPC is experimental and subject to change. Since
+  v1.5.0, ``torch.distributed.rpc`` supports calling TorchScript functions as
+  RPC target functions, and this will help improve parallelism on the callee
+  side as executing TorchScript functions does not require GIL. However, support
+  for calling RPC APIs within a TorchScript function is very limited.
```

Comment:
I would say we should even leave out mentioning support for calling RPC APIs within a TS function, because of the following reasons:

1) We only added rpc_async which was mostly to support an internal use case, and so it was likely designed as such, but we may need additional special attention for OSS
2) I don't believe we have any docs/tutorials mentioning which APIs are supported/unsupported
3) Support for things such as profiler for RPC called within JIT does not work, and there may be other features that don't work either. For example, are we able to call an async UDF via rpc_async inside a torchscript function? 
4) It hasn't yet been through a design/feature review which new features normally go through

Is it okay for now for it to be unannounced, and if users do discover it they can play around with it, but we harden and officially target 1.7 to release this?

Reply:
Good point, let me remove the last sentence.